lib/platform/git/storage.ts

```diff
@@ -125,6 +125,7 @@ export class Storage {
     const submodules = await this.getSubmodules();
     for (const submodule of submodules) {
       try {
+        logger.info(`Cloning git submodule at ${submodule}`);
```

Comment:
Do we need this? Can we lower the level?

Reply:
Debug would be nice